pkg/errors/errors.go

```diff
@@ -271,3 +274,8 @@ func IsClosedNetwork(err error) bool {
 func IsErrGitHubNotVerifiedEmail(err error) bool {
 	return err.Error() == ErrGitHubNotVerifiedEmail.Error()
 }
+
+// IsTimeout returns true if err is of the type timeout
+func IsTimeout(err error) bool {
```

Comment:
do we really need this func? Could we use errors.Is() directly?

Reply:
fair point, it had more logic before, but since I've changed to `errors.Is` I agree with you and it can be removed